packages/forms/src/Components/DateTimePicker.php

```diff
@@ -38,6 +38,8 @@ class DateTimePicker extends Field
 
     protected string | Closure | null $timezone = null;
 
+    protected string | Closure | null $icon = null;
```

Comment:
Should still accept `bool`?

Reply:
When you pass false `->icon(false)` then `$this->evaluate($this->icon)` returns empty string. 

I was trying to keep it simple without needing to add a `disableIcon` and simple blade if conditions.

Reply:
Hmm really? Evaluating `false` should still evaluate to `false`.
Also, the approach I mentioned is how we handle such cases in other places too.

Reply:
Sorry yeah that's what was throwing me - updated now